msu/files.py

```diff
@@ -1,16 +1,43 @@
 from urllib import parse
-from flask import current_app
 import boto3
 
+from flask import (
+    current_app,
+    flash,
+    request
+)
+
+
 s3_client = boto3.client('s3')
 
 def _url(bucket, loc_constraint, name):
     return 'https://%s.s3.%s.amazonaws.com/%s' % \
         (bucket, loc_constraint, name)
 
-def upload(key, data):
-    bucket = current_app.config['S3_BUCKET']
+def _upload(key, data, bucket_extension):
+
+    bucket = current_app.config[bucket_extension]
     constraint = s3_client.get_bucket_location(Bucket=bucket)['LocationConstraint']
     s3_client.upload_fileobj(data, bucket, key)
 
     return _url(bucket, constraint, key)
+
+def upload_file(key, data):
+    return _upload(key, data, 'S3_BUCKET_FILES')
+
+def upload_image(key,data):
+    return _upload(key, data, 'S3_BUCKET_IMAGES')
+    
+def check_file(request):
```

Comment:
You're mixing web and non web code together. `files.py` acts as an _abstraction_, and all it should do is handle file uploads. Here, not only are you mixing file handling code with request handling code (which is specific to the view it comes from!) but even adding specific error messages, messages which are UI elements and have nothing to do with the "business logic".

If you want to define a helper function, I would suggest making it in the file where it is relevant, `view.py` in this case.